src/phantom.js

```diff
@@ -75,9 +76,15 @@ export default class Phantom {
         this.process.on('error', error => {
             logger.error(`Could not spawn [${phantomjs.path}] executable. Please make sure phantomjs is installed correctly.`);
             logger.error(error);
+            this.kill('Process got an error');
```

Comment:
Oof my reading comprehension has gone out the door in this thread. Sorry about that guys. Yes, that should most certainly include the error